### PR TITLE
Update ERC-3643: Move to Last Call

### PR DESCRIPTION
**Pull request by Joachim-Lebrun**
copy of https://github.com/ethereum/EIPs/pull/7871

Move the ERC-3643 to Last Call status after 3 month of call for reviews from the community and successful review sessions with the [erc3643 association](https://www.erc3643.org/), where we passed in review with all the members of the association the whole description of the EIP and got approval from all to move forward to final status

### CODE REVIEW

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: ERCS/erc-3643.md
Comment (SamWilsn) [inline ERCS/erc-3643.md:8]:
```suggestion
last-call-deadline: 2023-11-20
```

If you give maintainers the permission to edit your branch, we can update your last call deadline just before merging.

**Comment by Joachim-Lebrun** [comment]
@SamWilsn i added a sub section in the rationale to address your concerns as discussed on the [Ethereum Magicians forum](https://ethereum-magicians.org/t/eip-3643-proposition-of-the-t-rex-token-standard-for-securities/6844/9?u=joachim-lebrun)  I hope this makes everything more clear

File: ERCS/erc-3643.md
Comment (Nakasar) [reply ERCS/erc-3643.md]: Hey! It seems we can't have this option because the fork is under a GitHub organization (@erc-3643) and not a personal account. We can add you as editor on the fork repo though.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**
